src/main/resources/data-repository-openapi.yaml

```diff
@@ -384,7 +384,8 @@ definitions:
         description: Id of the auto-generated default passthru dataset
       custodians:
         type: array
-          items: string
+        items:
+          type: string
```

Comment:
I think putting owner and custodian in here is too much overloading. We will have endpoints for controlling access. Let's not put them here.

We will know initial owner because it we do this in the context of an authenticated request. We shouldn't pass it in. Owner of a study is a permission that can be granted and revoked using the access control endpoints.

Reply:
👍 It's a good point and simplifies things.